### PR TITLE
lookup: add metadata on who to ping

### PR DESCRIPTION
**Pull request by gdams**
closes: https://github.com/nodejs/citgm/issues/359

##### Checklist

- [x] `npm test` passes
- [x] contribution guidelines followed [here](https://github.com/nodejs/citgm/blob/master/CONTRIBUTING.md)
- [x] commit message follows commit guidelines

### CODE REVIEW

**Comment by gdams** [comment]
Updated PTAL, I generally populated the list from using the top commiters but this may not always be correct

**Comment by gibfahn** [comment]
`meta data`->`metadata` in the commit message (I've fixed the title for you)

File: lib/lookup.json [hunk 1]
```diff
@@ -1,315 +1,396 @@
...
     "flaky": true,
-    "master": true
+    "master": true,
+    "maintainers": ["contra", "phated"]
   },
   "readable-stream": {
     "prefix": "v",
-    "flaky": ["ppc", "darwin"]
+    "flaky": ["ppc", "darwin"],
+    "maintainers": ["isaacs", "calvinmetcalf"]
```
Comment (addaleax) [inline lib/lookup.json]: @ nodejs/streams instead :)

File: lib/lookup.json [hunk 2]
```diff
@@ -1,315 +1,396 @@
...
+    "skip": true,
+    "maintainers": "alanshaw"
   },
   "eslint": {
     "prefix": "v",
     "flaky": ["v7", "s390", "ubuntu"],
     "skip": ["v8", "ppc"],
-    "expectFail": "fips"
+    "expectFail": "fips",
+    "maintainers": ["nzakas", "mysticatea"]
```
Comment (addaleax) [inline lib/lookup.json]: not sure but maybe @not-an-aardvark?

File: lib/lookup.json [hunk 3]
```diff
@@ -1,315 +1,396 @@
...
-    "flaky": ["win32", "ppc"]
+    "flaky": ["win32", "ppc"],
+    "maintainers": "indexzero"
   },
   "yargs": {
     "prefix": "v",
     "flaky": "aix",
-    "expectFail": "fips"
+    "expectFail": "fips",
+    "maintainers": "bcoe"
```
Comment (addaleax) [inline lib/lookup.json]: you can add me if you want (not a maintainer but would like the notifications :))

**Comment by gdams** [comment]
updated PTAL

**Comment by MylesBorins** [comment]
@gdams where did you get the names from?

**Comment by gdams** [comment]
@MylesBorins  I went to the module repo's and picked the top contributors

**Comment by MylesBorins** [comment]
This is super awesome! We can almost automate making issues when things fail now

**Comment by gdams** [comment]
that's what myself and @gibfahn were discussing

**Pull request merged**